lib/diagnostic.js

```diff
@@ -39,21 +50,33 @@ const owner = null
 // Methods: vocalize
 
 // Replace null with your answer
-const pet = null
+const pet = {
+  name: 'Leeroy',
+  species: 'dog',
+  breed: 'pitbull/shepard mix',
+  vocalization: function() {
+    console.log('woof, I am the best dog ever')
+  }
+}
 
 // 4. Create a constructor function for pet objects, as described above. The
 // constructor function should be responsible for setting the values of `name`,
 // `species`, `breed`, and `vocalization`.
 
 const Pet = function () {
-  /* Your answer here */
```

Comment:
Make sure this function accepts these parameters: `const Pet = function(name, species, breed, vocalization){...`